gammapy/irf/psf/map.py

```diff
@@ -135,15 +145,15 @@ def _get_irf_coords(self, **kwargs):
 
         return coords_irf
 
-    def containment(self, rad, energy_true, position=None):
+    def containment(self, rad, energy, position=None):
         """Containment at given coords
 
         Parameters
         ----------
         rad : `~astropy.units.Quantity`
             Rad value
-        energy_true : `~astropy.units.Quantity`
-            Energy true value
+        energy : `~astropy.units.Quantity`
+            Energy value
```

Comment:
Given that we generally call "energy" to the reco energy, I would specify in the docstring that this energy value could be true energy or reco energy depending on how the PSF is given

Reply:
But I see what you mean about the confusion in general with energy/energy true... I think the docstrings should be clearer that both types of energy are valid as input

Reply:
For clarity I introduced a second class PSFMapReco that redefine only the methods that take energy as argument to have a correct docstring